src/js/shield_defs.js

```diff
@@ -651,20 +638,37 @@ export function loadShields(shieldImages) {
     pillShield(Color.shields.white, Color.shields.blue, Color.shields.black),
     ["ETR"]
   );
-  shields["CA:ON:secondary"] = trapezoidDownShield;
+  shields["CA:ON:secondary"] = trapezoidShield(
+    -10,
+    Color.shields.white,
+    Color.shields.black
+  );
   shields["CA:ON:tertiary"] = roundedRectShield(
     Color.shields.white,
     Color.shields.black
   );
-  shields["CA:ON:Halton"] = trapezoidUpShieldGreenYellow;
-  shields["CA:ON:Peel"] = trapezoidUpShieldBlackYellow;
-  shields["CA:ON:Simcoe"] = {
-    ...trapezoidUpShield,
-    textColor: Color.shields.blue,
-    colorLighten: Color.shields.blue,
-  };
+  shields["CA:ON:Halton"] = trapezoidShield(
+    10,
```

Comment:
Should the angle default to 10°? It seems to be a pretty common design.

Reply:
I chose 10° for most of these because it was visually similar to the SVG icon, but these shields use different angles in real life. I don't want to set a precedent for all trapezoid shields to look the same just because they're trapezoids—if Nebraska state highways and Ontario county roads have different dimensions, we can try to match those dimensions if we want.

Also, negative numbers indicate a trapezoid with a long bottom (like `CA:ON:secondary`). The top-heavy trapezoid may be more common, and I did assign it a positive number accordingly, though it would feel weird to default to one or the other.